server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -71,28 +73,28 @@ protected AbstractServer(String appName, ConfigOpts opts, String[] args) {
         lmd.getIntervalMillis(context.getConfiguration()), TimeUnit.MILLISECONDS);
     ThreadPools.watchNonCriticalScheduledTask(future);
     processMetrics = new ProcessMetrics(context);
-    idleReportingPeriodNanos = TimeUnit.MILLISECONDS.toNanos(
+    idleReportingPeriod = Duration.ofMillis(
```

Comment:
Not a change for this PR, but looking at this I suspect adding a `Duration getDuration(Property)` to Accumulo configuration would improve this and other code.